WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/AddProductToOrder.swift

```diff
@@ -14,15 +14,39 @@ struct AddProductToOrder: View {
     var body: some View {
         NavigationView {
             Group {
-                List(viewModel.productRows) { viewModel in
-                    ProductRow(viewModel: viewModel)
-                }
-                .listStyle(PlainListStyle())
-                .renderedIf(viewModel.productRows.isNotEmpty)
+                switch viewModel.syncStatus {
+                case .results:
+                    List {
+                        ForEach(viewModel.productRows) { rowViewModel in
+                            ProductRow(viewModel: rowViewModel)
+                                .onAppear {
+                                    if rowViewModel == viewModel.productRows.last {
+                                        viewModel.syncNextPage()
+                                    }
+                                }
+                        }
 
-                EmptyState(title: Localization.emptyStateMessage, image: .emptyProductsTabImage)
-                .frame(maxHeight: .infinity)
-                .renderedIf(viewModel.productRows.isEmpty)
+                        // Infinite scroll indicator
+                        ProgressView()
+                            .frame(maxWidth: .infinity, alignment: .center)
+                            .listRowInsets(EdgeInsets())
+                            .listRowBackground(Color(.listBackground))
+                            .renderedIf(viewModel.shouldShowScrollIndicator)
```

Comment:
This view is loaded as the last item in the list, and unfortunately I can't find a way to remove the separator below the row except with `.listRowSeparator(.hidden)` which is iOS 15+.

Is it worth using an `if #available` check to add that modifier, and otherwise just show this (for devices with iOS <15)?

Reply:
I think yes, if it doesn't require much effort.

Reply:
Yep, this turned out to be a simple check, added in a325ea6.